frontend/src/components/GameCard/GameCard.tsx

```diff
@@ -13,32 +13,46 @@ interface GameCardProps {
 
 const GameCard: FC<GameCardProps> = ({ game }) => {
 	return (
-		<li className={styles.card}>
-			<h2>{game.name}</h2>
-			<p>{game.description}</p>
-			<p>Platform: {game.platformName}</p>
-			<p>Playtime: {game.playtimeMinutes} mins</p>
-			<p>Release date: {game.releaseDate}</p>
-			<p>
-				Status:
-				{game.isBorrowed ? 'Currently borrowed' : 'Currently Available'}
-			</p>
-			<p>Minimum players: {game.playerMin}</p>
-			<p>Maximum players: {game.playerMax}</p>
-			<p>Location: {game.location}</p>
-			<p>Owner: {game.owner}</p>
-
-			<PlayStatus game={game} />
-
-			<GameRating game={game} />
-
-			<BorrowGame game={game} />
-			<RemoveGame game={game} />
-		</li>
+		<Link href={`/game/${game.id}`} legacyBehavior>
```

Comment:
What's the reason to use `legacyBehavior` here?

Reply:
I didn't want it to be an `<a />` tag

Reply:
Had to test, and I see the issue why. Could be fixed in CSS, but this is cleaner.